index.d.ts

```diff
@@ -60,11 +68,21 @@ declare namespace electronDl {
 		Optional callback that receives an object containing information about the progress of the current download item.
 		*/
 		readonly onProgress?: (progress: Progress) => void;
+		
+		/**
+		Optional callback that receives an object containing information about the total progress of all download items.
```

Comment:
This needs to be clearer about its exact behavior. For example, progress for all downloads or just ones done through this package. How the progress work for multiple concurrent downloads. For example, if another download is started while one is running, the progress will jump backwards.